app/assets/javascripts/discourse/tests/fixtures/session-fixtures.js

```diff
@@ -14,6 +14,7 @@ export default {
       moderator: true,
       staff: true,
       can_create_group: true,
+      can_create_topic: true,
```

Comment:
This boolean has been present in the CurrentUserSerializer for a long time, but it was missing from this fixture